CDCgov/prime-reportstream
# FHIRpath error message, new AOE codes, and zip code fix

This PR contains various small fixes detailed below.

Test Steps:
1. To test fhirpath error message: run fhir path CLI tool and enter an invalid path such as `Bundle.entry.resource.ofType(MessageHeader).extension("https://reportstream.cdc.gov/fhir/StructureDefinition/msh-message-header").extension("MSH.7").value.changetimezone('America/Denver')`.  The log should display the specific FHIRLexer exception message.
2. To test zip code fix - run unit test 

## Changes
- log detailed message when evaluating fhirpath. Error messages come from `FHIRPathEngine.java` in fhirpath library
- trim hyphenated zip codes before doing a lookup on the ZipCode table
- add AOE codes that were missing in observation-mapping table (already pushed to both staging and prod)

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue
